sass/patternfly-react/_inline-edit.scss

```diff
@@ -0,0 +1,127 @@
+/**
+  Inline edit tables
+*/
+table.pf-table-inline-edit thead {
+  tr.editing {
+    th {
+      border-bottom-color: $table-border-hover;
+      border-bottom-width: 3px;
+    }
+  }
+}
+
+table.pf-table-inline-edit tbody {
+  tr {
+    td {
+      padding: 5px 10px 3px;
+    }
+
+    td.editable {
+      padding: 0;
+
+      &.editing {
+        padding: 2px 3px;
+      }
+    }
+  }
+
+  tr.editing {
+    td {
+      background: $table-bg-hover;
+      border-top-color: $table-border-hover;
+      border-bottom-color: $table-border-hover;
+
+      &:first-child {
+        border-left-color: $table-border-hover;
+      }
+
+      &:last-child {
+        border-right-color: $table-border-hover;
+      }
+    }
+  }
+
+  .input {
+    border: 1px solid transparent;
+    display: block;
+    margin: 2px 3px 2px;
+    min-height: 26px;
+    padding: 3px 6px 1px;
+
+    &:hover {
+      background: $color-pf-white;
+      border: 1px solid $input-border;
+      cursor: text;
+    }
+
+    &:hover::after {
+      content: "\f040";
+      float: right;
+      font-family: $icon-font-name-fa;
+      margin-left: 15px;
+    }
+  }
+
+  .static {
+    border: 1px solid transparent;
+    display: block;
+    margin: 2px 3px 2px;
+    padding: 3px 6px 1px;
+  }
+}
+
+.inline-edit-buttons {
+  background: $table-bg-hover;
+  border: 1px solid $table-border-hover;
+  margin: 0;
+  padding: 4px;
+  position: fixed;
+  z-index: 1000;
+
+  &.top {
+    border-bottom: 0;
+  }
+
+  &.bottom {
+    border-top: 0;
+  }
+
+  &.bold {
+    border-width: 3px;
+  }
+}
+
+/**
+  Inline forms
+*/
+.pf-inline-edit {
+  .value {
+    border: 1px solid transparent;
+    padding: 2px 6px;
+    display: inline-block;
+
+    &:hover {
+      border-color: $color-pf-black-400;
+    }
+
+    &.edit-icon-hover {
+      &:hover::after {
+        content: "\f040";
+        font-family: $icon-font-name-fa;
+        margin-left: 15px;
+      }
+    }
+
+    &.edit-icon-permanent {
+      &::after {
+        content: "\f040";
+        font-family: $icon-font-name-fa;
+        margin-left: 15px;
+      }
+    }
+  }
+
+  .spinner {
+    margin-left: 6px;
+  }
+}
```

Comment:
Need to add the less equivalent as well (until #169 gets implemented).

Reply:
Done